binary_transparency/firmware/internal/verify/bundle_test.go

```diff
@@ -24,12 +25,17 @@ import (
 )
 
 const (
-	goldenUpdate        = `{"FirmwareImage":"RmlybXdhcmUgaW1hZ2U=","ProofBundle":{"ManifestStatement":"eyJNZXRhZGF0YSI6ImV5SkVaWFpwWTJWSlJDSTZJbFJoYkd0cFpWUnZZWE4wWlhJaUxDSkdhWEp0ZDJGeVpWSmxkbWx6YVc5dUlqb3hMQ0pHYVhKdGQyRnlaVWx0WVdkbFUwaEJOVEV5SWpvaWFDOUtka293TlRFd1YwMU9RMXAyYWxkUFF6QlRVV292VWs5RVpUTXJTSE5GYTIwMGJVZE9lbFZNZEhCaFpWUnFkalpLV1N0M2NqUjVVMjVXZG5sR05UQmlRM0l4ZEhCeGQwTnZRV2htY0hGQ1dtd3lZbEU5UFNJc0lrVjRjR1ZqZEdWa1JtbHliWGRoY21WTlpXRnpkWEpsYldWdWRDSTZJbWd2U25aS01EVXhNRmROVGtOYWRtcFhUME13VTFGcUwxSlBSR1V6SzBoelJXdHRORzFIVG5wVlRIUndZV1ZVYW5ZMlNsa3JkM0kwZVZOdVZuWjVSalV3WWtOeU1YUndjWGREYjBGb1puQnhRbHBzTW1KUlBUMGlMQ0pDZFdsc1pGUnBiV1Z6ZEdGdGNDSTZJakl3TWpBdE1URXRNalJVTVRFNk5UZzZNRGxhSW4wPSIsIlNpZ25hdHVyZSI6IlRFOU1JUT09In0=","Checkpoint":{"TreeSize":94,"RootHash":"uVxL27F8Cn//fR540AOXYUAytI0Zzy3q6EsuIY6W+mM=","TimestampNanos":1606219089169612444},"InclusionProof":{"Value":null,"LeafIndex":93,"Proof":["Kv2X4qnhywOdFuygRzEfrp9r+1+6+oLeW0DqaFTl2ZQ=","EJWrh9mfnUYB2k8jdhLH47qAELCSBn993oBrQ2i/I1Q=","D3+q/Q0ZYUtWz1vTr90bmaZoF/Q/VJsScGiAYGG3M9g=","jAJirzErZ93Fa9qi2UCJ++6ygGY4X+jDRcukr0dmt0U=","+IVLLpscVZdckOLOnobkEx99wkHFIC1vhvIiWzFqkDM="]}}}`
+	goldenUpdate        = `{"FirmwareImage":"RmlybXdhcmUgaW1hZ2U=","ProofBundle":{"ManifestStatement":"eyJNZXRhZGF0YSI6ImV5SkVaWFpwWTJWSlJDSTZJbVIxYlcxNUlpd2lSbWx5YlhkaGNtVlNaWFpwYzJsdmJpSTZNU3dpUm1seWJYZGhjbVZKYldGblpWTklRVFV4TWlJNkltZ3ZTblpLTURVeE1GZE5Ua05hZG1wWFQwTXdVMUZxTDFKUFJHVXpLMGh6Uld0dE5HMUhUbnBWVEhSd1lXVlVhblkyU2xrcmQzSTBlVk51Vm5aNVJqVXdZa055TVhSd2NYZERiMEZvWm5CeFFscHNNbUpSUFQwaUxDSkZlSEJsWTNSbFpFWnBjbTEzWVhKbFRXVmhjM1Z5WlcxbGJuUWlPaUkyZEZWWGVYbHJlbmRtYjI5dVIzVm5ibVl4WkV3clkyZGtObUpGVjFWb2IyeEJUbUZFVERoS1dVdFhkR1J0VUdORlpWbDJaMDgyYmsxeUwwbE1aMWRRWTFWWloyZDJRVUZ5Y25SaFUwSnRZVzQxU0RSTFp6MDlJaXdpUW5WcGJHUlVhVzFsYzNSaGJYQWlPaUl5TURJd0xURXdMVEV3VkRFMU9qTXdPakl3TGpFd1dpSjkiLCJTaWduYXR1cmUiOiJFaXBNMXRMdjF4cnJMSHZEdC80VDFHUG9KV3hBYlExMmhiMkZTOWQ1cDhsbjBKeWJJZFBieVBPWTVYMXozbVBUV0VnMnp1VTB1aWs0VmQwNW84dmM1cGRPZEFTSHlCeDA5RXBhT0NjTWVxSW9SRm90N3lvVUVDdUxkZHBCNEw4aWlEZ3Vibnk1Tk8zaTkzTjNFcnBUclN0b1ZqWjd1ZnZRd082SWg4aWpQZTVTY0o5TG1zQjBMRkZKeUIvQVNnYXcyeE9NWDVnMjlxSzR5UWNBak11WlE3b25ITG95Z09pK2pWUy92akJ0SEVxcXQ1RVU3dU9NdVJVSitqOFYva25yWUJya2hMNEVqWW9SZFNKTnZ6azVpMDRrdGNWLzJQb1NBR2RqSi9rejMrUG1idStXUjRRMVZMcng2bzBaVFNRMi94dXR2K1d2K0lmQXFOdDB0QldoWnc9PSJ9","Checkpoint":{"TreeSize":5,"RootHash":"4E7J8K809jeqeg1oiTIz+5zfMItqZqUBFR0jySa3H/M=","TimestampNanos":1607450738111506088},"InclusionProof":{"Value":null,"LeafIndex":4,"Proof":["KFh4IVeIwbsvbWyz2QHVCXXyjWTRDqusRa0ZEjS2fls="]}}}`
 	goldenFirmwareImage = `Firmware image`
+	// b64 is a base64 encoded string for ExpectedMeasurement field inside ManifestStatement.
+	// For the dummy device, this is SHA512("dummy"||img)
```

Comment:
maybe add "where `img` is the base64 decoded bytes from goldenUpdate.FirmwareImage" to be super clear